osu.Game.Rulesets.Osu/Mods/OsuModSnapColour.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
+
```

Comment:
No `#nullable disable` in new code
https://github.com/ppy/osu/wiki/Nullable-Reference-Types-(NRT)#dependency-injection-considerations

Reply:
Thank you very much for the quick code review, I removed #nullable disable and made your suggested fixes